android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/comment/display/CommentDisplayView.java

```diff
@@ -87,6 +87,7 @@ private void loadView(boolean editable) {
 			enableEditionButtons(editable);
 			showEditionButtons();
 			userPortraitScreenlet.setUserId(commentEntry.getUserId());
+			userPortraitScreenlet.load();
 			userNameTextView.setText(commentEntry.getUserName());
```

Comment:
Maybe we can check here if the current userId and the userportrait's current userId are the same and not load the portrait? what do you think?